### DISCUSSION TITLE
Is the repo and project still actively maintained?

### DISCUSSION THREAD

**Discussion by mxa0079** [nx-dotnet/nx-dotnet#851] [General]
I see no commits for 3 months and simple PRs that have not been merge since February.

Is the core team still committed to this project? I understand some of you work directly at NX.

Please advice, as we have a production code base that depends on this plugin.

**Comment by AgentEnder** [comment]
Hey - yes the plugin is still maintained. Its worth noting that the "core team" for this plugin is just me. There's only so much time to go around, and as-is, the plugin is functional and working. I'll make sure it stays that way, but I don't have a ton of time to work on new features. 

Typically you'll see more activity here around the time frame of a new major Nx version, as that's when I'm re-affirming compatibility and more likely to make small tweaks.

**Reply by mxa0079** [reply]
Thanks @AgentEnder - how can we help out as consumers of the plugin? Is NX also committed to this plugin?

**Reply by AgentEnder** [reply]
There are two main pieces of work in flight. 

- v19 PR. This one's gotten a bit out of hand if we are honest, as I've rewrote how a lot of the generator code works from scratch at a fundamental level. The result should be much more accurate from Nx's POV but it's making the process take longer.

- Swagger updates. NSwag's CLI tool to pull swagger doc from your built assembly isn't functional for latest .NET versions. We need to find a way to handle this, as in the v19 PR I'm just skipping the tests. This is also why the smoke tests are red.

**Reply by mxa0079** [reply]
@AgentEnder how are things going? This plugin is crucial for us  - how can we help you? It seems a bit daunting to start contributing without your help. I'd love to jump on a call or 1:1 chat to see if there is a way for us to get more involved.

Thanks for your help!
